pytripgui/controller/settings_cont.py

```diff
@@ -0,0 +1,100 @@
+import os
+import pickle
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SettingsController:
+    """
+    Class for interacting with saved configurations.
+    """
+
+    def __init__(self, model):
+        """
+        """
+
+        self.model = model
+        # new settings handler here
+
+        self.pyc_path = os.path.join(self.get_user_directory(), "settings.pyc")
+        logger.debug("New setup config parser based on {:s}".format(self.pyc_path))
+
+        self.load()
+
+    def load(self):
+        """
+        Load confif, sets self.model accordingly.
+        If file does not exist, exit silently.
+
+        It is connected to Model, in a way that
+        - upon SettingsController.load(), SettingsModel attributes starting with "_"  are not written to         Model.
+        - upon SettingsController.save(),         Model attributes starting with "__" are not written to SettingsModel.
+
+        This way,
+            a) _version is written to disk, but imported into Model when loading
+            b) __internal_attribute__ are not passed between Model and SettingsModel
+        """
+
+        logger.debug("SettingsController.load() : loading settings from {}".format(self.pyc_path))
+
+        model = self.model
+        pyc = self.pyc_path
+
+        if os.path.isfile(pyc):
+            with open(pyc, 'rb') as f:
+                _ms = pickle.load(f)
+
+            # set all model attributes accordingly
+            for _attr in dir(_ms):
+                if _attr[0] != "_":  # do not copy internal variables
+                    _value = getattr(_ms, _attr)
+                    setattr(model, _attr, _value)
+        else:
+            logger.info("Settings file {} not found.".format(pyc))
+
+    def save(self, path=None):
+        """
+        Saves the current model configuration to the settings file.
```

Comment:
The docstring is repeated twice, if written in such way it should be moved to class or module level.